packages/vaadin-combo-box/test/overlay-position.test.js

```diff
@@ -79,7 +80,7 @@ describe('overlay position', () => {
       comboBox.open();
       await aTimeout(1);
       comboBox.items = [1, 2, 3];
-      await aTimeout(1);
+      await nextRender(comboBox.$.overlay);
```

Comment:
Same as above, use `comboBox.$.dropdown` instead. Also, `nextRender` is `requestAnimationFrame` + `setTimeout` and here we had `setTimeout` - is this more reliable? Which one should we use?